packages/dashboard-grid-layout/src/components/DashboardGridLayout/DashboardGridLayout.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import { Responsive, WidthProvider, Layout, Layouts } from 'react-grid-layout';
+import {
+  Size,
+  IDashboardGridLayoutProps,
+  IDashboardGridLayoutStyles,
+  IDashboardCardLayout
+} from './DashboardGridLayout.types';
+import { getStyles } from './DashboardGridLayout.styles';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+const ResponsiveReactGridLayout = WidthProvider(Responsive);
+
+export class DashboardGridLayout extends React.Component<IDashboardGridLayoutProps, {}> {
+  private rowHeight = 50;
```

Comment:
Don't use privates for constants, unless you plan on mutating the values.

Instead use file level consts. (move above the class.)

```tsx
const RowHeight = 50;
const Breakpoints = { ... };
```